src/Sulu/Bundle/AdminBundle/Resources/js/index.js

```diff
@@ -130,6 +130,9 @@ Requester.handleResponseHooks.push(logoutOnUnauthorizedResponse);
 
 initializeJexl();
 
+export const ERROR_CODE_DEPENDANT_RESOURCES_FOUND = 1105;
+export const ERROR_CODE_REFERENCING_RESOURCES_FOUND = 1106;
```

Comment:
These constants can't be placed in this `index.js` file, because that file may only be imported once, because otherwise all calls like `initializer.addUpdateConfigHook()` would be run twice

Reply:
makes sense - good to know 🙂